File: cases/taint/python/argument-position-negative/argument_position.py
def dfb_source():  # DFB-SOURCE: argument-position-negative-input
    return "tainted"


def choose_first(first, second):  # DFB-WITNESS: argument-position-negative-first
    return first


def dfb_sink(value):  # DFB-SINK: argument-position-negative-sink
    pass


def run():
    result = choose_first("clean", dfb_source())
    dfb_sink(result)

File: cases/taint/ruby/argument-position-negative/argument_position.rb
def dfb_source # DFB-SOURCE: argument-position-negative-input
  "tainted"
end

def choose_first(first, second) # DFB-WITNESS: argument-position-negative-first
  first
end

def dfb_sink(value) # DFB-SINK: argument-position-negative-sink
end

def run
  result = choose_first("clean", dfb_source)
  dfb_sink(result)
end
